content/docs/communities/administration/_index.md

```diff
@@ -0,0 +1,112 @@
+    - only aliases on the same homeserver will be transferred automatically
+    - consider romo v12 creator semantics: use a service account the access to which you can control outside of the rules of matrix
+    - if your community is using a canonical space, the one upgrading the room needs at least enough access to the parent space to update the reference from the old to the upgraded room
+  - if your old room was published in the public room directory, you might have to update the reference to the new room manually
+  - plan with rate limits
+    - servers will only allow you to create so many new rooms within a certain time, so you might get stuck half-way if you have many rooms. a workaround would be to use a special account that your server admin has excepted from this rate limiting. for example, your moderation bot account is a good candidate for this.
+    - servers will only allow you to send a certain amount of invites as well as only allow users to join proactively at a certain rate
+    - if your community has a lot of rooms that are being upgraded, you might consider sending follow-up pings to the old rooms to make sure everyone got notified about every room, since it is easy to lose a room due to the different notification and rate limiting mechanics in play
+  - remind your users that they may need to "complete" the update on their side
+    - e.g. their non-canonical spaces, room directories, external aliases
+  - send a message linking to the new room into the old room to support clients without support for room upgrades (see list below). you might consider pining the message, using a room ping, etc.
+
+### Limitations and things to consider
+
+- Matrix does not tightly define what clients or servers do when using the room upgrade function (link to spec)
+- ...
+- Rate limits
+  - for executing upgrades (room creation)
+  - for inviting users
+  - for users (re-)joining proactively
+- Upgrading spaces
+- Upgrading rooms in spaces
+  - spaces you own
+  - spaces others own you know nothing about
+- Moving external aliases that may not be visible to you
+- Moving over integrations with external references to the room (bridges? Moderation bots? Webhooks?)
+- Migrating certain room state (e.g. custom state events)
+- Certain room state getting carried over unintentionally (e.g. bans)
+- Restricted room join references? Does anybody care about those?
+- Published Room directory
+  - on other servers
+  - when the room directory is not writable by users
+- Cover clients that do not support tombstones -> need to send a link, pin, etc in the old room that links to the new one
+- Making old rooms read-only
+  - Some clients make rooms read-only when they see a tombstone even when technically that's not the case
+- Matrix.org premium accounts
+  - free users will be able to upgrade public rooms (<https://matrix.to/#/!sWpnrYUMmaBrlqfRdn:matrix.org/%24G6iZU-2eJu5QQgdxu5yMX2aXza642UKMCxyvLjpMMEI?via=matrix.org&via=t2l.io&via=element.io>)
```

Comment:
Update to reference https://github.com/matrix-org/matrix.org/pull/2862 when merged

Reply:
```diff
- - free users will be able to upgrade public rooms (<https://matrix.to/#/!sWpnrYUMmaBrlqfRdn:matrix.org/%24G6iZU-2eJu5QQgdxu5yMX2aXza642UKMCxyvLjpMMEI?via=matrix.org&via=t2l.io&via=element.io>)
+ - free users will be able to upgrade public rooms using the [upgrade room API](https://spec.matrix.org/v1.15/client-server-api/#post_matrixclientv3roomsroomidupgrade). Manually creating the new room and later manually creating a tombstone in in the old room is not possible. See the [Matrix.org Homeserver pricing page](@/homeserver/pricing.md#can-an-account-on-the-free-usage-plan-upgrade-a-public-room) for more details.
```